src/main/java/com/redislabs/WriteSource.java

```diff
@@ -212,6 +217,9 @@ public boolean isWriteThrough() {
   public int getTimeout() {
     return timeout;
   }
+  public boolean getWriteOnChange() {
```

Comment:
Can we add a setter as well for serialisation and deserialisation

Reply:
Added setter